hudi-utilities/src/main/java/org/apache/hudi/utilities/schema/KafkaOffsetPostProcessor.java

```diff
@@ -43,7 +45,7 @@ public static class Config {
         HoodieStreamerConfig.KAFKA_APPEND_OFFSETS;
 
     public static boolean shouldAddOffsets(TypedProperties props) {
-      return props.getBoolean(HoodieStreamerConfig.KAFKA_APPEND_OFFSETS.key(), Boolean.parseBoolean(HoodieStreamerConfig.KAFKA_APPEND_OFFSETS.defaultValue()));
+      return getBooleanWithAltKeys(props, HoodieStreamerConfig.KAFKA_APPEND_OFFSETS);
```

Comment:
not required in this patch. 
but we should have two apis. one which return default if not found. another which returns null 

```
getBooleanWithAltKeys
getBooleanWithAltKeysOrDefault 
```

Reply:
For `getBooleanWithAltKeys` it would be overkill.  For String value, I added different flavors of `getStringWithAltKeys`.